pkg/common/common.go

```diff
@@ -28,15 +29,22 @@ import (
 
 // TODO: move the const to a proper place, or get it from config
 const (
-	KuadrantNamespace                    = "kuadrant-system"
 	KuadrantRateLimitClusterName         = "kuadrant-rate-limiting-service"
 	HTTPRouteKind                        = "HTTPRoute"
 	KuadrantRateLimitPolicyRefAnnotation = "kuadrant.io/ratelimitpolicies"
 	RateLimitPolicyBackRefAnnotation     = "kuadrant.io/ratelimitpolicy-direct-backref"
 	AuthPolicyBackRefAnnotation          = "kuadrant.io/authpolicy-backref"
+	KuadrantNamespaceLabel               = "kuadrant.io/namespace"
 	NamespaceSeparator                   = '/'
+	LimitadorName                        = "limitador"
```

Comment:
Did you mean to remove the ability to redefine this thru a environment variable?

Reply:
Yes, I could rollback to env var with default too.

Reply:
I just thought if there's only one limitador and authorino per kuadrant, there was no need to make a difference